src/components/CodeInput/CodeInput.tsx

```diff
@@ -49,7 +50,13 @@ const CodeInput: React.FC<Props> = (props: Props): ReactElement => {
   return (
     <div className={classnames(STYLE.wrapper, className)} data-level={messageType}>
       <VerificationInput
-        inputProps={{ 'aria-label': ariaLabel, id: inputId, name: inputId, disabled }}
+        inputProps={{
+          'aria-label': `${ariaLabel ? ariaLabel : ''}, field ${value.length + 1} of ${numDigits}`,
+          'aria-describedby': `${ariaDescribedby}`,
```

Comment:
nitpicking: the template string is not necessary
```suggestion
          'aria-describedby': ariaDescribedby,
```